### ISSUE TITLE
Fast switching window using Alt+Tab trigger fallback mode

### ISSUE THREAD

**Issue by erusyd** [linuxmint/cinnamon#11792]
### Distribution

Mint 21.2

### Package version

Cinnamon 5.8.4

### Graphics hardware in use

NVIDIA RTX A4000, NVIDIA GeForce GTX 940MX, and Intel GPU

### Frequency

Always

### Bug description

In U++ (https://www.ultimatepp.org/), after clicking "New package", continuously fast switching window using Alt+Tab will trigger fallback mode. I tested U++ under manjaro-xfce, mint-xfce and windows 10, and different laptops, no such problem.

### Steps to reproduce

 (1) start TheIDE and open an example, then continuously fast switching window by Alt+Tab won't trigger fallback (2) "File"->"Open main package", then continuously fast switching window won't trigger fallback (2) Click "New package", then continuously fast switching window using Alt+Tab after many times trigger fallback
![record](https://github.com/linuxmint/cinnamon/assets/1529102/e5228372-863d-4e49-a36a-338c671b3024)

### Expected behavior

Should not trigger fallback.

**Comment by magnite-dom** [comment]
Noticed a similar issue on a Latitude 7430 switching between text editor and terminal

Distro: Mint 21.2 upgraded from 21.1
Package Ver: Cinnamon 5.8.4

Hardware: 
i7-1265U
Intel Graphics

**Comment by mtwebster** [comment]
Please list your alt-tab settings:
![image](https://github.com/linuxmint/cinnamon/assets/262776/bd0583f8-54a4-49cf-a522-c5699d025e7e)

**Comment by magnite-dom** [comment]
> 

I switched to "Icons Only" after the crashes happened, before that I was at the default
![image](https://github.com/linuxmint/cinnamon/assets/132091069/24102e84-8b14-4294-a625-4f455d5019e0)

**Comment by erusyd** [comment]
I think I use the default:
![image](https://github.com/linuxmint/cinnamon/assets/1529102/6d8cde88-a18c-46b1-b84f-f012dfdda275)

**Comment by erusyd** [comment]
I change switcher style to "Icons only", continuously fast switching U++ "New package" window still triggers fallback mode. You can try to see if it happens on your computer (keep fast switching continuously).

**Comment by magnite-dom** [comment]
> I change switcher style to "Icons only", continuously fast switching U++ "New package" window still triggers fallback mode. You can try to see if it happens on your computer (keep fast switching continuously).

Yeah I switched to icons only thinking it may help being the least demanding option. I havent had a crash since however I feel like I have also been more reserved when alt+tabbing to avoid triggering this.

**Comment by mtwebster** [comment]
Fixed by e2cb99ad4ab4f729571791e1c4337f7e3850fdf6
